### ISSUE TITLE
Does not run

### ISSUE THREAD

**Issue by UltraBlackLinux** [trigg/Discover#86]
After running Discover from the terminal, I discovered, it needs pidfile. Aftrer installing it throws some more errors:
```
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/discover_overlay/discover_overlay.py", line 127, in entrypoint
    with pidfile.PIDFile(pid_file):
AttributeError: module 'pidfile' has no attribute 'PIDFile'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/discover-overlay", line 33, in <module>
    sys.exit(load_entry_point('discover-overlay==0.2.2', 'console_scripts', 'discover-overlay')())
  File "/usr/lib/python3.8/site-packages/discover_overlay/discover_overlay.py", line 130, in entrypoint
    except pidfile.AlreadyRunningError:
AttributeError: module 'pidfile' has no attribute 'AlreadyRunningError'
```

Please fix that, I want to use Discover!

**Comment by midget3111** [comment]
Hey, couple of questions.
What OS?
What Desktop environment?
How did you install & how are you trying to run Discover?

**Comment by UltraBlackLinux** [comment]
Linux
KDE
AUR, Running via the terminal (doesn't really matter, its the same always)

**Comment by midget3111** [comment]
Thanks, please try running the latest testing version and see whether that works.

**Comment by UltraBlackLinux** [comment]
still the same.

**Comment by dsjoho** [comment]
Did you install python-pid and change the import statement? I did that at first on Arch, since I found python-pid but not python-pidfile.
I.e. https://github.com/trbs/pid/ instead  of https://github.com/mosquito/python-pidfile

It runs on Arch using python-pid, if you open discover_overlay.py and replace the import pidfile with "import pid", and in entrypoint():
pidfile.PIDFile --> pid.PidFile
pidfile.AlreadyRunningError --> pid.base.PidFileAlreadyRunningError
A total change of 3 rows (or something like that, just fix the errors and it'll run afterwards). But this is of course a different package, and it doesn't look like any of them have been dropped.
